src/test/java/uk/gov/companieshouse/mapper/ReasonMapperTest.java

```diff
@@ -1,24 +1,32 @@
 package uk.gov.companieshouse.mapper;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.continuedIllness;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.illPerson;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.illnessEnd;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.illnessImpactFurtherInformation;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.illnessStart;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.IllnessReason.otherPerson;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.OtherReason.description;
+import static uk.gov.companieshouse.TestData.Appeal.Reason.OtherReason.title;
+
+import java.util.Collections;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.companieshouse.database.entity.IllnessReasonEntity;
 import uk.gov.companieshouse.database.entity.OtherReasonEntity;
 import uk.gov.companieshouse.database.entity.ReasonEntity;
+import uk.gov.companieshouse.model.IllnessReason;
 import uk.gov.companieshouse.model.OtherReason;
 import uk.gov.companieshouse.model.Reason;
 
-import java.util.Collections;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static uk.gov.companieshouse.TestData.Appeal.Reason.OtherReason.description;
-import static uk.gov.companieshouse.TestData.Appeal.Reason.OtherReason.title;
-
 @ExtendWith(SpringExtension.class)
 public class ReasonMapperTest {
-    private final ReasonMapper mapper = new ReasonMapper(new OtherReasonMapper(new AttachmentMapper()));
+    private final ReasonMapper mapper = new ReasonMapper(new OtherReasonMapper(new AttachmentMapper()),
+        new IllnessReasonMapper(new AttachmentMapper()));
```

Comment:
Same comment as the AppealsMapperTest class - using Mockito would be simpler and more readable.

Reply:
Resolved on https://github.com/companieshouse/lfp-appeals-api/pull/112